lmnet/lmnet/datasets/pascalvoc_2007.py

```diff
@@ -47,8 +47,11 @@ class Pascalvoc2007(PascalvocBase):
 
     @property
     def num_max_boxes(self):
-        # calculate by cls.count_max_boxes()
-        return 42
+        # alculate by cls.count_max_boxes(self.skip_difficult)
```

Comment:
`calculate` is misspelled.

Reply:
fix https://github.com/blue-oil/blueoil/pull/107/commits/85285191de51e44297e3922c701e0a4863f4189f